client/app/src/Controller/Report/ReportController.php

```diff
@@ -208,7 +208,9 @@ public function clientHomepageAction(Redirector $redirector, string $clientId)
             'clientHasCoDeputies' => $this->preRegistrationApi->clientHasCoDeputies($clientWithCoDeputies->getCaseNumber()),
         ];
 
-        if ($ndrEnabled) {
+        $ndrExistsOnClient = $this->restClient->get('/ndr/client/'.$clientId, 'array');
```

Comment:
I think we should be able to test this on the Client instance without requiring a separate API call.